pkg/agent/proxy/proxier.go

```diff
@@ -618,66 +636,73 @@ func (p *proxier) installServiceFlows(svcInfo *types.ServiceInfo, internalGroupI
 	}
 
 	// Install ClusterIP flows.
-	if err := p.ofClient.InstallServiceFlows(internalGroupID, svcInfo.ClusterIP(), uint16(svcInfo.Port()), svcInfo.OFProtocol, affinityTimeout, false, isNestedService); err != nil {
+	if err := p.ofClient.InstallServiceFlows(internalGroupID, clusterGroupID, svcInfo.ClusterIP(), svcPort, svcProto, affinityTimeout, false, isNestedService); err != nil {
```

Comment:
should we just use 0 as clusterGroupID here to make it explicit, and easier to predict the value in tests?

Reply:
Updated as you said.